README.md

```
@@ -1,337 +1,288 @@
+
+2. **Instale as dependências**
+   ```bash
+   flutter pub get
+   ```
+
+3. **Execute a aplicação**
+   ```bash
+   flutter run
+   ```
+
+### **Build para Produção**
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)